lib/season.rb

```diff
@@ -4,13 +4,11 @@ def initialize(game_team_data, game_data, team_data)
     @game_team_data = game_team_data
     @game_data = game_data
     @team_data = team_data
-    @games_data = games_data
   end
 
-
   def most_tackles(season)
     game_ids = []
-    @games_data.each do |row|
+    @game_data.each do |row|
```

Comment:
I will respect this team decision on naming conventions. Glad we talked about it.